carma/launch/environment.launch.py

```diff
@@ -90,7 +90,8 @@ def generate_launch_description():
     carma_wm_ctrl_param_file = os.path.join(
         get_package_share_directory('carma_wm_ctrl'), 'config/parameters.yaml')
 
-
+    carma_cooperative_perception_param_file = os.path.join(
+        get_package_share_directory('carma_cooperative_perception'), 'config/params.yaml')
```

Comment:
The recommended way to do paths now is with `pathlib`. This should be refactored to something like this:
```python
carma_cooperative_perception_param_file = str(get_package_share_directory("carma_cooperative_perception") / "config/params.yaml")
```

Reply:
imported pathlib to handle this